arelle/plugin/validate/EFM/Filing.py

```diff
@@ -2741,14 +2742,12 @@ def addCoverFactRecord(facts):
                 jsonParam[_objName] = _orderedObjFacts
             if isFeeTagging:
                 jsonObjType = "fee"
-                messageCode = "EFM.feeFacts"
                 testEnvJsonFile = val.params.get("saveFeeFacts")
             else:
                 jsonObjType = "cover"
-                messageCode = "EFM.coverFacts"
                 testEnvJsonFile = val.params.get("saveCoverFacts")
             modelXbrl.log("INFO-RESULT",
-                          messageCode,
+                          "EFM.{}Facts".format(jsonObjType),
```

Comment:
This reverts #1221 and should be undone.

Reply:
added in messageCodes=("EFM.feeFacts", "EFM.coverFacts") which is used in the other places where dynamic constructs generate multiple message codes.